code/integration-tests/local-integration-tests/src/helpers.rs

```diff
@@ -20,13 +20,14 @@ pub fn para_account_id(id: u32) -> AccountId {
 }
 
 /// under ED, but above Weight
-pub fn under_existential_deposit<AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>>(
+pub fn under_existential_deposit<
+	AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>
+		+ AssetExistentialDepositInspect<AssetId = CurrencyId, Balance = Balance>,
+>(
 	asset_id: LocalAssetId,
 	_instruction_count: usize,
 ) -> Balance {
-	PriceConverter::<AssetsRegistry>::get_price_inverse(asset_id, NativeExistentialDeposit::get())
-		.unwrap() /
-		Balance::from(2_u128)
+	multi_existential_deposits::<AssetsRegistry>(&asset_id) / 2_u128
```

Comment:
What's the magic number 2 here?

Reply:
it ensures the name of method to be true 'under_existential_deposit'